docs/best-practices/pingone/index.md

```diff
@@ -0,0 +1,248 @@
+# Terraform Writing Best Practices - PingOne
+
+The following provides a set of best practices to apply when writing Terraform with the PingOne Terraform provider and associated modules.
+
+These guidelines do not intend to educate on the use of Terraform, nor are they a getting started guide.  For more information about Terraform, visit [Hashicorp's Online Documentation](https://developer.hashicorp.com/terraform/docs).  To get started with Ping Identity Terraform providers, visit the online [Getting Started](./../index.md) guides.
+
+## Develop in the Admin Console, Promote using Configuration-As-Code
+
+Ping recommends that use case development activities are performed in the PingOne web admin console wherever possible.  This is due to the complex nature of Workforce IAM and Customer IAM deployments that includes policy definition, user experience design and associated testing/validation of designed use cases.
+
+Once developed in the web admin console, configuration can be extracted as configuration-as-code to be stored in source control (such as a Git code repository) and linked with CI/CD tooling to automate the delivery of use cases into test and production environments.
+
+For professionals experienced in DevOps development, configuration may be created and altered outside of the web admin console, but care must be taken when modifying complex configuration such Authorize, MFA, Protect or SSO sign-on policies.
+
+## Example / Bootstrapped Configuration Dependencies
+
+### Deploy to "Clean" Environments, without Example / Bootstrapped Configuration
+
+Example / bootstrapped configuration is deployed automatically by the PingOne service when an environment is created (or new services are provisioned to an existing environment).  This is the default behaviour of the web admin console, and the API.
+
+Example / bootstrapped configuration may be useful as a starting point when initially creating use cases with the service (in the development phase), but will create conflicts when migrating the configuration through to test and production environments.
+
+The definition of the example / bootstrapped configuration for new environment may also change over time, as new features are released and use case configuration best practices are defined.  Therefore an environment created today may not be the same as an environment created a year from now.
+
+It is best practice therefore to, when creating new environments other than the initial development environment, create them as "clean" environments, without example / bootstrapped configuration.  Currently, example / bootstrapped configuration must be manually removed from an environment.
```

Comment:
As a result, it is best practice to create a new environment as a "clean" (without example or bootstrapped configuration) environment for those environments outside of the initial development one.